cluster-up/cluster/images.sh

```diff
@@ -6,8 +6,8 @@ declare -A IMAGES
 IMAGES[gocli]="gocli@sha256:220f55f6b1bcb3975d535948d335bd0e6b6297149a3eba1a4c14cad9ac80f80d"
 if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0@sha256:aebf67b8b1b499c721f4d98a7ab9542c680553a14cbc144d1fa701fe611f3c0d"
-    IMAGES[k8s-1.18]="k8s-1.18@sha256:906af927cd0a998166d12c61162db2a0eb395c9e65d537ce7d1af2d603cb8690"
-    IMAGES[k8s-1.17]="k8s-1.17@sha256:c60f8450457d0678c40247abaad6a687cc6cfe1c1ac307291ae4008f1c490240"
+    IMAGES[k8s-1.18]="k8s-1.18@sha256:c41e3d9adb756b60e1fbce2ffd774c66c99fdde7ee337460c472ee92868e579e"
```

Comment:
just wondering, what was introduced in those?
its not sriov, but i understand it should be part of the bump

Reply:
ah its ok, because i fixed k8s provisioning, and changed just cluster-provision and this file from cluster-up folder

Reply:
I used `make bump-kubecitci` so its pure automatic ;)

Reply:
yep all good, (i assume you used make generate afterwards), i am trying to resolve the need of it, in other PR
thanks